examples/chef/nrfconnect/main.cpp

```diff
@@ -87,9 +87,7 @@ CHIP_ERROR main()
     chip::app::DnssdServer::Instance().SetExtendedDiscoveryTimeoutSecs(kExtDiscoveryTimeoutSecs);
 
     // Start IM server
-    static chip::CommonCaseDeviceServerInitParams initParams;
-    (void) initParams.InitializeStaticResourcesBeforeServerInit();
-    ReturnErrorOnFailure(chip::Server::GetInstance().Init(initParams));
+    ReturnErrorOnFailure(chip::Server::GetInstance().Init());
```

Comment:
Please note that this is an interop branch and this changes were made because it wasn't building before.

Init on this interop branch doesn't have any parameters.